From 705c84321daeb847cecba7f1e1f8f3e23f8165f8 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 12 May 2014 14:36:08 +0200
Subject: [PATCH] cms/cmssetup.sh: convert SUBCHANNELS to lowercase

In udev rules, the channels need to be expressed in lower case.

(cherry picked from commit 56d57a70ec89740dc6ddc09ea23b6fc4c3a6a531)
---
 modules.d/80cms/cmssetup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/80cms/cmssetup.sh b/modules.d/80cms/cmssetup.sh
index 36955bc..c9e1331 100755
--- a/modules.d/80cms/cmssetup.sh
+++ b/modules.d/80cms/cmssetup.sh
@@ -144,6 +144,7 @@ function dasd_settle_all() {
     processcmsfile()
     {
         source /tmp/cms.conf
+        SUBCHANNELS="$(echo $SUBCHANNELS | sed 'y/ABCDEF/abcdef/')"
 
         if [[ $NETTYPE ]]; then
 	    (
